lib_eio/flow.mli

```diff
@@ -62,8 +62,12 @@ type read_method += Read_source_buffer of ((Cstruct.t list -> int) -> unit)
 class virtual sink : object
   inherit Generic.t
   method virtual copy : 'a. (#source as 'a) -> unit
+  method virtual write_from : Cstruct.t -> unit
 end
 
+val write : #sink -> Cstruct.t -> unit
+(** [write dst buf] writes one or more bytes from [buf] *)
```

Comment:
Might be good to note here that this is a low-level API and that using `copy` may allow optimisations in some cases.